lib/livebook_web/live/session_live/insert_buttons_component.ex

```diff
@@ -8,29 +8,36 @@ defmodule LivebookWeb.SessionLive.InsertButtonsComponent do
       aria-label="insert new"
       data-element="insert-buttons">
       <div class={"w-full absolute z-10 focus-within:z-[11] #{if(@persistent, do: "opacity-100", else: "opacity-0")} hover:opacity-100 focus-within:opacity-100 flex space-x-2 justify-center items-center"}>
-        <button class="button-base button-small"
-          phx-click="insert_cell_below"
-          phx-value-type="markdown"
-          phx-value-section_id={@section_id}
-          phx-value-cell_id={@cell_id}
-          >+ Markdown</button>
         <button class="button-base button-small"
           phx-click="insert_cell_below"
           phx-value-type="code"
           phx-value-section_id={@section_id}
           phx-value-cell_id={@cell_id}
           >+ Code</button>
-        <button class="button-base button-small"
-          phx-click="insert_section_below"
-          phx-value-section_id={@section_id}
-          phx-value-cell_id={@cell_id}
-          >+ Section</button>
-        <%= if @smart_cell_definitions != [] do %>
-          <.menu id={"#{@id}-smart-cell-menu"}>
-            <:toggle>
-              <button class="button-base button-small">+ Smart</button>
-            </:toggle>
-            <:content>
+        <.menu id={"#{@id}-insert-menu"} position="left">
+          <:toggle>
+            <button class="button-base button-small">+ Block</button>
+          </:toggle>
+          <:content>
+            <button class="menu-item text-gray-500"
+              role="menuitem"
+              phx-click="insert_cell_below"
+              phx-value-type="markdown"
+              phx-value-section_id={@section_id}
+              phx-value-cell_id={@cell_id}>
+              <.remix_icon icon="markdown-fill" />
+              <span class="font-medium">Markdown</span>
+            </button>
+            <button class="menu-item text-gray-500"
+              role="menuitem"
+              phx-click="insert_section_below"
+              phx-value-section_id={@section_id}
+              phx-value-cell_id={@cell_id}>
+              <.remix_icon icon="h-2" />
+              <span class="font-medium">Section</span>
+            </button>
+            <%= if @smart_cell_definitions != [] do %>
+              <div class="h-[0.5px] bg-gray-200"></div>
```

Comment:
Can you add a bit of space top and bottom of the line? Other than that, ship it!